docs/architecture/decisions/0004-generating-unique-ids.md

```diff
@@ -0,0 +1,47 @@
+# 1. Record architecture decisions
+
+Date: 2023-03-16
+
+## Status
+
+Accepted
+
+## Context
+
+When users start an LPA either online or via the paper route, their data will be sent to and stored in this ID service data store. We need to create a unique ID to store alongside the user's data, which case workers can then use to find LPA's by when actioning any future enquiries or objections.
```

Comment:
it would be helpful to avoid abbreviations in case they mean different things in different contexts. In this case, ID.